### DISCUSSION TITLE
dp.Select / dropdown not working for long item list

### DISCUSSION THREAD

**Discussion by BenjaminNSD** [datapane/datapane#168] [Help]
Datapane is great. I just encountered some problems with selecting items in a dropdown list.

`dp.Select(type=dp.SelectType.DROPDOWN`

 When the item list is long I only can select the first few items from the list. I cannot scroll down to the buttom of the list, the dropdown window is closing then

I've installed datapane 0.12.1

**Comment by BenjaminNSD** [comment]
It's actually working on my work pc with bog screen, but not on my little notebook

**Comment by mands** [comment]
Thanks @BenjaminNSD for raising the issue - we'll take a look and test on smaller screen sizes / resolutions

**Comment by Buroni** [comment]
Hey @BenjaminNSD I'm trying to reproduce this locally, and am unable to so far. Can you provide your browser, OS and an example of a report where the issue occurs? Thanks!
